src/redux/configureStore.js

```diff
@@ -1,16 +1,14 @@
-import React, { Component } from 'react';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
 
-class configureStore extends Component {
-  constructor() {
-    super();
-    this.state = {};
-  }
+const reducer = combineReducers({
+  booksReducer,
+});
```

Comment:
- [ ] This is required as a change because I can confirm from previous reviews that you had previously imported the `categoryReducer` here and added it to the `combineReducers`. Kindly re-import the `categoryReducer` and add it to the `reducer` here, just as you had it before. This is important for subsequent projects.